.github/pull_request_template.md

```diff
@@ -1,18 +0,0 @@
-## Objective
-
-
-## Detailed Description
-
-
-## Screenshot (if applicable)
-
-
-## How to Test (if applicable)
-
-
-## Tests (if added)
-
-
-## Requirements
-[Taiga User Story](https://tree.taiga.io/project/cbrobsto-iron-db/us/xx)
-[Taiga Task](https://tree.taiga.io/project/cbrobsto-iron-db/task/xx)
```

Comment:
Any idea why this would've been deleted when you moved `Express.js` to the API folder?

Reply:
I have no idea, actually! I'll try and get that fixed really quickly.

Reply:
No worries.

Reply:
It also looks like the `.gitignore` and topline `Readme.md` files were moved into the `API` directory as well.

Reply:
Yeah, I think we'll need to create new ones for the top level. The `client` and `API` folders can each have their own `README`s and `.gitignores`. It might be better to have that stuff added before we merge. I'm starting to think we'd be better off waiting until the next sprint to merge this so that we aren't putting in something incomplete.

Reply:
Sounds good. We can discuss this more during retro, but it would be really helpful to get a CI/CD setup going for unit & ui tests so that we're not missing anything on these PRs.